src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -59,6 +60,8 @@ public static Index parseIndex(String oneBasedIndex) throws ParseException {
             }
             indexesList[i] = Index.fromOneBased(Integer.parseInt(trimmedIndex));
         }
+        // Sort the array from large to small so that invalid (out of range) index will be encountered first
```

Comment:
Maybe do you want to add into the Javadocs the behaviour of the parseMultipleIndex now?